File: week01-210619-210625/moregorenine/stages/s1_IO_and_arithmetic_operations/q10_a_remain_b/main.py
import sys

number1, number2, number3 = map(int, sys.stdin.readline().split())

print((number1 + number2) % number3)
print(((number1 % number3) + (number2 % number3)) % number3)
print((number1 * number2) % number3)
print(((number1 % number3) * (number2 % number3)) % number3)

File: week01-210619-210625/moregorenine/stages/s1_IO_and_arithmetic_operations/q10_a_remain_b/main.js
const fs = require('fs')
const stdin = (process.platform === 'linux'
        ? fs.readFileSync('/dev/stdin').toString()
        : `5 8 4`
).split(' ')

const number1 = parseInt(stdin[0], 10)
const number2 = parseInt(stdin[1], 10)
const number3 = parseInt(stdin[2], 10)

console.log((number1 + number2) % number3)
console.log(((number1 % number3) + (number2 % number3)) % number3)
console.log((number1 * number2) % number3)
console.log(((number1 % number3) * (number2 % number3)) % number3)